### PR TITLE
chore: remove selected overrides

### PR DESCRIPTION
**Pull request by pavelfeldman**

### CODE REVIEW

File: src/Playwright/Request.cs [hunk 1]
```diff
@@ -72,7 +72,7 @@ internal Request(IChannelOwner parent, string guid, RequestInitializer initializ
 
         public async Task<IResponse> ResponseAsync() => (await _channel.GetResponseAsync().ConfigureAwait(false))?.Object;
 
-        public JsonDocument PostDataJSON(JsonDocumentOptions documentOptions = default)
```
Comment (kblok) [inline src/Playwright/Request.cs:75]: I consider this important to users. Without this argument, this method will be useless  to many times.

File: src/Playwright/Contracts/IPage.cs [hunk 2]
```diff
@@ -8,9 +8,6 @@ namespace Microsoft.Playwright
 {
     public partial interface IPage
     {
-        /// <inheritdoc cref="EmulateMediaAsync(Media?, ColorScheme?)"/>
-        Task EmulateMediaAsync(ColorScheme? colorScheme);
```
Comment (kblok) [inline src/Playwright/Contracts/IPage.cs:12]: Are tests running without this?

Comment (pavelfeldman) [reply src/Playwright/Contracts/IPage.cs:12]: They compiled...

File: src/Playwright/Request.cs (hunk 1 above)
Comment (pavelfeldman) [reply src/Playwright/Request.cs:75]: I'm looking at the properties and they seem to be for lax parsing, which we don't need. MaxDepth - we should hard-code to -1, i'll do it.

**Pull request merged**